web/src/wasm.ts

```diff
@@ -207,14 +208,21 @@ export class Backend {
   }
 
   getDemandModel(): FeatureCollection<MultiPolygon, ZoneDemandProps> {
-    return JSON.parse(this.inner.getDemandModel());
+    let gj = JSON.parse(this.inner.getDemandModel());
+    for (let f of gj.features) {
```

Comment:
An example where this WASM wrapper layer can do a bit of work, instead of having to do it on the Rust side